core/internal/cltest/cltest.go

```diff
@@ -239,13 +234,12 @@ func NewEthConfirmer(t testing.TB, db *gorm.DB, ethClient eth.Client, config evm
 type TestApplication struct {
```

Comment:
To be honest, it might be easier just to get rid of this entire struct (not in this PR tho)